src/client/GlobalData.ts

```diff
@@ -12,14 +12,16 @@
 /**
  * Populate and export global data
  */
-
 import { signal } from "@preact/signals";
 
+import bliss_symbols from "../../public/data/bliss_symbol_explanations.json";
```

Comment:
As we discussed yesterday, this line leads to a warning message. Can you have a look on how easy or hard to fix it? Although it's a warning, I'm worried once we live with it now, if it causes a failure in the future, we need to re-investigate this whole thing again. Let me know what you think. Thanks.

Reply:
Sure at some point.  One fact to note is that this warning only happens when serving up one of the demo apps using the vite server.  It does not happen using express.  The first thing I'd look into is whether serving the demo apps with express would eliminate the warning.

Reply:
As we discussed, I added a comment about the warning from the `vite` server here.